OpenRA.Mods.Common/Traits/Render/WithBuildingPlacedOverlay.cs

```diff
@@ -15,7 +15,7 @@
 namespace OpenRA.Mods.Common.Traits.Render
 {
 	[Desc("Rendered when the actor constructed a building.")]
-	public class WithBuildingPlacedOverlayInfo : ITraitInfo, Requires<RenderSpritesInfo>, Requires<BodyOrientationInfo>
+	public class WithBuildingPlacedOverlayInfo : ConditionalTraitInfo, Requires<RenderSpritesInfo>, Requires<BodyOrientationInfo>
```

Comment:
Can we please drop this change from the PR?

We don't need it for the hotfix, and it will come along with many other related changes in #15682.